draft-ietf-teep-protocol.xml

```diff
@@ -227,32 +227,91 @@ otherwise.
 
     <section anchor="detailmsg" title="Detailed Messages Specification">
 
-    <t>The CBOR-encoded messages are protected by COSE, as described in CDDL format <xref target="I-D.ietf-cbor-cddl"/> below. 
+    <t>TEEP messages are protected by the COSE_Sign1 structure.
+      The TEEP protocol messages are described in CDDL format <xref target="I-D.ietf-cbor-cddl"/> below. 
     
  <figure>
  <artwork><![CDATA[
```

Comment:
This edit looks incomplete.  Please check the CDDL after changing it...

Reply:
More CDDL edits will follow from Akira